### PR TITLE
Localisation updates from https://translatewiki.net.

### PR DESCRIPTION
**Pull request by niedzielski**

### CODE REVIEW

File: Wikipedia/Localizations/br.lproj/Localizable.strings [hunk 1]
```diff
@@ -331,7 +331,7 @@
 "home-nearby-check-again" = "Gwiriañ en-dro";
 "home-main-page-heading" = "Hiziv e Wikipedia";
 "home-potd-heading" = "Skeudenn an deiz evit $1";
-"home-last-update-label" = "Hizivadenn diwezhañ: $1";
+"home-last-update-label" = "Hizivadenn diwezhañ: %1$";
```
Comment (niedzielski) [inline Wikipedia/Localizations/br.lproj/Localizable.strings]: I wasn't sure if this is right or not. I see some similar %1 elsewhere:

```
Wikipedia/Localizations/vi.lproj/Localizable.strings
259:"page_protected_other" = "Trang này đã bị khóa ở các mức độ sau: %1";

Wikipedia/Localizations/sq.lproj/Localizable.strings
116:"main-menu-current-article-save" = "Ruani %1 për lexim pa internet";

Wikipedia/Localizations/hu.lproj/Localizable.strings
192:"page_protected_other" = "Az a lap az alábbi szinten védett: %1";

Wikipedia/Localizations/tr.lproj/Localizable.strings
265:"page_protected_other" = "Bu sayfa aşağıdaki seviyelerde korunmaktadır: %1";
```

File: Wikipedia/Localizations/cs.lproj/Localizable.strings [hunk 2]
```diff
@@ -181,19 +184,29 @@
...
 "share-menu-save-page" = "Uložit článek";
 "share-menu-page-saved" = "Článek uložen";
 "share-custom-menu-item" = "Sdílet";
-"share-on-twitter-sign-off" = "via @Wikipedia";
+"share-on-twitter-sign-off" = "via @Wikipedie";
 "share-as-image" = "Sdílet jako obrázek";
 "share-as-text" = "Sdílet jako text";
 "share-cancel" = "Zrušit";
 "timestamp-just-now" = "právě teď";
+"timestamp-minutes" = "{{PLURAL:%d|%d minuta|%d minuty|%d minut}}";
```
Comment (niedzielski) [inline Wikipedia/Localizations/cs.lproj/Localizable.strings]: Probably not right?

File: Wikipedia/Localizations/cs.lproj/Localizable.strings [hunk 3]
```diff
@@ -181,19 +184,29 @@
 "about-repositories" = "Repozitáře";
 "about-repositories-app-source-license" = "Zdrojový kód je k dispozici pod licencí $1.";
 "about-repositories-app-source-license-mit" = "Licence MIT";
+"about-send-feedback" = "Poslat názor na aplikaci";
 "about-wikimedia-foundation" = "Nadace Wikimedia";
 "share-menu-save-page" = "Uložit článek";
 "share-menu-page-saved" = "Článek uložen";
 "share-custom-menu-item" = "Sdílet";
-"share-on-twitter-sign-off" = "via @Wikipedia";
+"share-on-twitter-sign-off" = "via @Wikipedie";
```
Comment (niedzielski) [inline Wikipedia/Localizations/cs.lproj/Localizable.strings]: Seemed appropriate to me.

File: Wikipedia/Localizations/br.lproj/Localizable.strings (hunk 1 above)
Comment (montehurd) [inline Wikipedia/Localizations/br.lproj/Localizable.strings]: @niedzielski good catch! adding test to catch this in the future...

**Comment by montehurd** [comment]
👍 

@coreyfloyd should be good

**Pull request merged**